tests/ui/layout/debug.stderr

```diff
@@ -1,4 +1,4 @@
-error: layout_of(E) = Layout {
+error: layout_of(Adt(E, [])) = Layout {
```

Comment:
I feel like the layout tests should actually just use the Display impl for Ty...

Reply:
Good point. If I use `Display` for `normalized_ty` then we go back to the old, better output.